src/api_recievers/influxqueries.js

```diff
@@ -0,0 +1,204 @@
+// const { time } = require("console");
+// const { start } = require("repl");
+
+const { currentTime, InfluxDB, Point } = require("@influxdata/influxdb-client");
+
+const log = reuqire('../util/logger');
+
+
+const token = process.env.INFLUXDB_TOKEN;
+const url = 'http://localhost:8086';
+const client = new InfluxDB({url, token});
+let org = `Distraction`;
+let bucket = `WebsiteData`;
+
+
+//Write Function
+//Source is from Chrome API or OS 
+//Must provide the application_name
+//Must provide the current study session this entry is associaed with
+//Inputs are Strings
+const appData = (source, appName, currentSession) =>{
+  let writeClient = client.getWriteApi(org, bucket, 's');
+  
+  let app = new Point('AppChange').stringField('AppName',appName).timestamp(currentTime.seconds()).stringField('Source', source).tag('QuerySession',currentSession);
```

Comment:
We should try to keep each of these on their own line so it's easier to read:
```
let app = new Point('AppChange')
    .stringField('AppName',appName)
    .timestamp(currentTime.seconds())
    .stringField('Source', source)
    .tag('QuerySession',currentSession);
```

Reply:
Done